src/components/InventoryV2/InventoryListByNursery.tsx

```diff
@@ -52,7 +52,11 @@ const columns = (): TableColumnType[] => [
   { key: 'totalQuantity', name: strings.TOTAL, type: 'string', tooltipTitle: strings.TOOLTIP_TOTAL_QUANTITY },
 ];
 
-export default function InventoryListByNursery() {
+type InventoryListByNurseryProps = {
```

Comment:
here's a `type` :) 
i have mostly been using `type` myself... we should decide on one for sure. Not sure if there's a preference among us.

Reply:
Oh, I searched in the whole project without the "_=_", that's why I only got interfaces in my search result. But you are right, we have a mix of both. I can change the interface to be type if that's our preference